05_syntax_lab/01.py

```diff
@@ -3,3 +3,14 @@
 the user and prints the largest one
 """
 
+largestNumber = int(0)
+
+for n in range(10):
+    print"please enter number(%d)" %(n+1)
+    currentnumber=int(raw_input())
+    if largestNumber < currentnumber:
+        largestNumber = currentnumber
+
+
+
+print('largest number is :'),largestNumber  
```

Comment:
A more pythonic print would probably use str.format:
`print 'largest number is: {}'.format(largestNumber)`